### DISCUSSION TITLE
DATA FOR NSE(indian Markets) UNAVAILABLE

### DISCUSSION THREAD

**Discussion by psachdev42** [ranaroussi/yfinance#2089] [General]
Hi, 

When i try to fetch data for symbols related to NSE, example(SBIN, RELIANCE). Empty dataframe is returned. is there a restriction on exchange? is someone working on this?

**Comment by ericpien** [comment]
Try adding `.NS` or `.BO` at the end for Indian markets. The ticker has to reflect what's shown on Yahoo.

```Python
import yfinance as yf
tickers = yf.Tickers(['SBIN.NS','RELIANCE.NS','SBIN.BO'])
tickers.download()
```

**Reply by psachdev42** [reply]
Hi Eric, 

This helps , Thankyou.

also is there a limit on the number of tickers we can query at a time?
